docs/subnets/add-subnet.md

```diff
@@ -17,13 +17,12 @@ required_permissions:
 1. Find the `Networks` card
 1. Click `View` in the `Networks` card, to see a list of existing `Networks`
 1. In the list of networks find the network where you want to add a subnet and click the `NAME` of the network
-1. On the upper-right corner of network details view, click `SUBNETS`
-1. On the upper-right corner of subnets list, click `ADD SUBNET`
-1. Click `NEXT`
+1. On the upper-right corner of network details view, click the `Subnets` tab
+1. On the upper-right corner of subnets tab click `MANAGE`
+1. Follow the prompts
 1. Provide a name to the Subnet being added. Example: "Public Subnet A", "Private Subnet B"
 1. Select a `Type` and click `NEXT`
-1. Follow the prompts
 1. Provide a CIDR block to the Subnet being added. Example: "10.108.0.0/20"
-1. Click `NEXT`
 1. Click `CONFIRM`
-1. In the `Subnets` list, the recently created `Subnet` can be seen
+    - For the new `Subnet`, it will be started the provision action
+1. In the `Subnets` tab, the recently created `Subnet` can be seen
```

Comment:
After creating a subnet user is redirected to action viewer running `subnet:provision` action, therefore there is no "subnets tab" to be seen, thus this tab should not be mentioned here.
```suggestion
```